src/App.tsx

```diff
@@ -1,61 +1,118 @@
 import './App.scss';
+import { FC, useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList';
+import { Todo } from './components/Types/Todo';
+
+import {
+  findUserById,
+  preparedTodos,
+  newTodoId,
+} from './components/Helpers/helpers';
+
+export const App: FC = () => {
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [isTitleError, setisTitleError] = useState(false);
+  const [isUserError, setisUserError] = useState(false);
+  const [todos, setTodos] = useState(preparedTodos);
+
+  const submitForm = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (!title) {
+      setisTitleError(true);
+    }
+
+    if (!selectedUser) {
+      setisUserError(true);
+    }
+
+    if (!title || !selectedUser) {
+      return;
+    }
+
+    const newTodo: Todo = {
+      id: newTodoId,
+      title: title.trim(),
+      completed: false,
+      userId: Number(selectedUser),
+      user: findUserById(Number(selectedUser)),
+    };
+
+    setTodos([...todos, newTodo]);
+    setTitle('');
+    setSelectedUser(0);
+  };
+
+  const changeTitle = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
good practice to name the event handler as handle[eventName]:smiley:

`handleTitleChange`
`handleUserChange`
`handleSubmit`